src/internal/operators/catchError.ts

```diff
@@ -14,8 +14,18 @@ export function catchError<T, O extends ObservableInput<any>>(selector: (err: an
 /**
  * Catches errors on the observable to be handled by returning a new observable or throwing an error.
  *
+ * <span class="informal">It only listens to the error channel and ignores notifications.
+ * If an error is emitted it catches it and retrieves the error as parameter.
+ * It can either return a new `Observable` to swallow the error or rethrow an new error.</span>
+ *
  * ![](catch.png)
  *
+ * The `catchError` operator is here to handle errors within the pipe function.
+ * The operator only listens on the error channel. It completely ignores next or complete events.
+ * If an error is received it can d 2 different things. Catching the error or rethrow a new error.
+ * The error is received as parameter of the operator and the return value of the operator is a new `Observable`
+ * that either rethrows a new error or returns a new Observable that maybe emits a value and maybe completes.
```

Comment:
Again, here I'm not sure of the explanation.

```
This operator handles errors, but forwards along all other events to the resulting observable. If the source observable terminates with an error, it will map that error to a new observable, subscribe to it, and forward all of its events to the resulting observable.
```